tests/test_pipelines_automatic_speech_recognition.py

```diff
@@ -215,3 +215,20 @@ def test_xls_r_from_en(self):
         filename = ds[40]["file"]
         output = speech_recognizer(filename)
         self.assertEqual(output, {"text": "Ein Mann sagte zu dem Universum, Sir, ich bin da."})
+
+    @slow
+    @require_torch
+    @require_torchaudio
+    def test_wav2vec2_2_bart(self):
```

Comment:
Is this test here intentional, using wav2vec2 (not generative) leads me to believe it isn't intentional.

Regardless, `pipeline(model="patrickvonplaten/wav2vec2-2-bart-base")` should work out of the box FYI.